content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -2,34 +2,34 @@
 After-Sales services
 ====================
 
-*After-Sales* services can be configured in the *Helpdesk* application for individual *teams*. Once
+*After-Sales* services can be configured in the *Helpdesk* application for individual teams. Once
 enabled, users can issue refunds, process returns, generate coupons, and/or schedule repair and
```

Comment:
Just an idea- these can all link to anchors in the doc. So the refunds takes you to the refund section, etc.